.github/workflows/build-pr-preview.yml

```diff
@@ -219,7 +219,8 @@ jobs:
           cd unified-docs-frontend-preview
           echo "HASHI_ENV=unified-docs-sandbox" >> .env
           echo "UNIFIED_DOCS_API=${{ needs.deploy-unified-docs-api-preview.outputs.preview_url }}" >> .env
-
+          echo "VERCEL_AUTOMATION_BYPASS_SECRET=${{ secrets.VERCEL_AUTOMATION_BYPASS_SECRET }}" >> .env
```

Comment:
can you add this env var to the project settings for the frontend-preview too?

Reply:
Done.